wkcuber/wkcuber/convert_raw.py

```diff
@@ -23,7 +23,7 @@
 logger = logging.getLogger(__name__)
 
 
-CHUNK_SIZE = 1024, 1024, 1024
+BLOCK_SIZE = 32
```

Comment:
@normanrz There's also `DEFAULT_WKW_FILE_LEN` in `webknossos.dataset.defaults`, as well as `BLOCK_LEN`
and `DEFAULT_WKW_VOXELS_PER_BLOCK = 32` in `wkcuber.utils`.

Reply:
ah, ok. then `DEFAULT_WKW_VOXELS_PER_BLOCK` should probably be used here instead.

Reply:
Perfect! I could have grabbed this from `convert_nifti.py`.